pkg/clientaccess/token.go

```diff
@@ -287,7 +307,13 @@ func (i *Info) Put(path string, body []byte) error {
 	}
 	p.Scheme = u.Scheme
 	p.Host = u.Host
-	return put(p.String(), body, GetHTTPClient(i.CACerts, i.CertFile, i.KeyFile), i.Username, i.Password, i.Token())
+
+	client := GetHTTPClient(i.CACerts, i.CertFile, i.KeyFile, option...)
+	for _, o := range option {
+		o(client)
+	}
+
+	return put(p.String(), body, client, i.Username, i.Password, i.Token())
```

Comment:
Clean this up since you moved the option stuff into GetHTTPClient

```suggestion
	return put(p.String(), body, GetHTTPClient(i.CACerts, i.CertFile, i.KeyFile, option...), i.Username, i.Password, i.Token())
```

Reply:
My bad, need to slow down on fixes.